web/src/engine/providers/MangaPlugin.ts

```diff
@@ -25,7 +25,7 @@ export abstract class MangaScraper extends MediaScraper<MangaPlugin> {
         return new MangaPlugin(storageController, settingsManager, this);
     }
 
-    public abstract ValidateMangaURL(url: string): boolean;
+    public abstract ValidateMangaURL(url: string): Promise<boolean>;
```

Comment:
The `ValidateMangaURL` is intended to be as fast as possible and therefore must not be async (this method will be called for all websites whenever an URL is pasted from clipboard)